interp/notation.ml

```diff
@@ -2103,18 +2113,30 @@ let split_notation_string str =
     if i < String.length str then
       if str.[i] == ' ' then
         push_token beg i (loop_on_whitespace (i+1) (i+1))
+      else if beg = i && str.[i] = '"' then
+        loop_on_string i (i+1)
       else
         loop beg (i+1)
     else
       push_token beg i []
   and loop_on_whitespace beg i =
     if i < String.length str then
       if str.[i] != ' ' then
-        push_whitespace beg i (loop i (i+1))
+        push_whitespace beg i (loop i i)
       else
         loop_on_whitespace beg (i+1)
     else
       push_whitespace beg i []
+  and loop_on_string beg i =
+    if i < String.length str then
+      if str.[i] = '"' then
+        if i+1 < String.length str then
+          if str.[i+1] = '"' then loop_on_string beg (i+2)
+          else if str.[i+1] = ' ' then push_token beg (i+1) (loop_on_whitespace (i+2) (i+2))
+          else user_err (Pp.str "End of string not followed by a space in notation.")
```

Comment:
This error message (and the one below it) should probably be documented

Reply:
Done